conf_parse.go

```diff
@@ -98,12 +98,20 @@ func confFromStr(conf string) (*lineSenderConfig, error) {
 				return nil, NewInvalidConfigStrError("invalid %s value, %q is not 'on' or 'off'", k, v)
 			}
 		case "auto_flush_rows":
+			if v == "off" {
```

Comment:
Could you also update the godoc for `LineSenderFromConf`, so that it mentions the "off" values?